distribution/ddf-common/src/main/resources/etc/transformers/ddf-default-input-transformers.json

```diff
@@ -0,0 +1,14 @@
+[
+  "confluence",
+  "gmd:MD_Metadata",
```

Comment:
Is this valid JSON even though its not wrapped in `{}`?

Reply:
Yes.